Driverless AI/risk-assessment-tools-in-the-world-of-ai-the-social-sciences-and-humanities/risk-assessment-tools-in-the-world-of-ai-the-social-sciences-and-humanities.md

```diff
@@ -0,0 +1,475 @@
+
+**Problem 4: Defining Fairness**
+
+The AI world lacks a concrete moral definition of what will be considered an excellent fair model. The author's option is that by studying the several existing moral philosophies, we will clearly define fairness in the AI world. That is why more interdisciplinary conversations around AI and the social sciences and humanities are necessary. If we want to integrate AI into several aspects of our daily lives, we need to make sure that such AI will not create or amplify social problems. That is why we need a concrete definition of fairness to force every AI model to abide by the fairness criteria.   
+
+**Problem 5: Prior Criminal Counts**
+
+As discovered, it is clear that someone's prior criminal count can't be a clear reflection of true recidivism. Like prior counts, we also notice that age is another factor that shouldn't be considered when predicting recidivism. Correspondingly, we need to replace these traditionally used recidivism factors and explore for new unbiased features. Thus, we need joint work between AI, the social sciences, and the humanities if we hope to predict multiple recidivism types truly. 
+
+**Final Analysis**
+
+As a final analysis, the AI world is growing and changing our society. We need the social sciences and humanities to solve the problems AI is creating during its creation. This joint work can genuinely be the key to more acceptance of AI into our daily lives while leveraging AI's power. Most of the errors produced by AI models are impacting the most marginalized groups. This dislike needs to change, and the change starts when we work together to understand the several impacts these models can have on the most marginalized groups. Whether people are already integrating the social sciences and humanities to AI, we need to make sure that such integration is speedup because we currently have the wrong following titles in several news sources across the country: 
+
+- When AI in healthcare goes wrong, who is responsible?
+- Rise of the racist robots – how AI is learning all our worst impulses
+- Racist Data? Human Bias is Infecting AI Development
+- Google apologizes after its Vision AI produced racist results
+- 600,000 Images Removed from AI Database After Art Project Exposes Racist Bias
+
+### References 
+
+[9][When a Computer Program Keeps You in Jail](https://www.nytimes.com/2017/06/13/opinion/how-computers-are-harming-criminal-justice.html)
+
+[10][Confrontation Clause - Wikipedia](https://en.wikipedia.org/wiki/Confrontation_Clause)
+
+## Next Steps
```

Comment:
The next steps for this tutorial should point to other references related to this tutorial since we don't have the next tutorial to follow this one. This is technically tutorial 5B in the MLI section.

Reply:
So because this is technically the last tutorial in that section we should eliminate the Next Steps section for now?